manifest.json

```diff
@@ -590,6 +590,30 @@
                 "UploadPartCopy"
             ]
         },
+        "S3Vectors": {
+            "source": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/s3vectors/2025-07-15/api-2.json",
+            "documentation": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/s3vectors/2025-07-15/docs-2.json",
+            "pagination": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/s3vectors/2025-07-15/paginators-1.json",
+            "waiter": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/s3vectors/2025-07-15/waiters-2.json",
+            "example": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/s3vectors/2025-07-15/examples-1.json",
+            "api-reference": "https://docs.aws.amazon.com/AmazonS3/latest/API/API_Operations_Amazon_S3_Vectors.html",
+            "methods": [
+                "CreateIndex",
+                "CreateVectorBucket",
+                "DeleteIndex",
+                "DeleteVectorBucket",
+                "DeleteVectorBucketPolicy",
+                "DeleteVectors",
+                "GetIndex",
+                "GetVectorBucket",
+                "GetVectorBucketPolicy",
+                "ListIndexes",
+                "ListTagsForResource",
+                "ListVectorBuckets",
+                "PutVectorBucketPolicy",
+                "TagResource"
```

Comment:
Do you need all these operations?

The philosophy of this project is to add only what is asked/needed by people, not to add everything.